docs/apidocs/index.rst

```diff
@@ -1,14 +1,13 @@
-.. _aqctensor:
-
-.. module:: qiskit_addon_aqc_tensor
-
-========================
-AQC-Tensor API Reference
-========================
+=========================================
+``qiskit-addon-aqc-tensor`` API Reference
+=========================================
 
 .. toctree::
    :maxdepth: 1
 
+   qiskit-addon-aqc-tensor
```

Comment:
If we remove the only function from this page (as done in d6b74c8546cbde495cccbf0), should the page still exist?

Reply:
No, the page should be deleted. We can add it back later if we end up wanting to document things here

Reply:
I will not be home for two hours. Feel free to make this edit or I can when I'm back.